docs/docs/examples/llm/ipex_llm_gpu.ipynb

```diff
@@ -0,0 +1,182 @@
+    "\n",
+    "If you are a Linux user, visit the [Install IPEX-LLM on Linux with Intel GPU](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/install_linux_gpu.html), and follow [**Install Prerequisites**](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/install_linux_gpu.html#install-prerequisites) to install GPU driver, Intel® oneAPI Base Toolkit 2024.0, and Conda.\n",
+    "\n",
+    "## Install `llama-index-llms-ipex-llm`\n",
+    "\n",
+    "After the prerequisites installation, you should have created a conda environment with all prerequisites installed, activate your conda environment and install `llama-index-llms-ipex-llm` as follows:\n",
+    "\n",
+    "```bash\n",
+    "conda activate <your-conda-env-name>\n",
+    "\n",
+    "pip install llama-index-llms-ipex-llm[xpu] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/\n",
+    "```\n",
+    "This step will also install `ipex-llm` and its dependencies.\n",
+    "\n",
+    "> **Note**\n",
+    ">\n",
+    "> You can also use `https://pytorch-extension.intel.com/release-whl/stable/xpu/cn/` as the `extra-indel-url`.\n",
+    "\n",
+    "\n",
+    "## Runtime Configuration\n",
+    "\n",
+    "For optimal performance, it is recommended to set several environment variables based on your device:\n",
+    "\n",
+    "### For Windows Users with Intel Core Ultra integrated GPU\n",
+    "\n",
+    "In Anaconda Prompt:\n",
+    "\n",
+    "```\n",
+    "set SYCL_CACHE_PERSISTENT=1\n",
+    "set BIGDL_LLM_XMX_DISABLED=1\n",
+    "```\n",
+    "\n",
+    "### For Linux Users with Intel Arc A-Series GPU\n",
+    "\n",
+    "```bash\n",
+    "# Configure oneAPI environment variables. Required step for APT or offline installed oneAPI.\n",
+    "# Skip this step for PIP-installed oneAPI since the environment has already been configured in LD_LIBRARY_PATH.\n",
+    "source /opt/intel/oneapi/setvars.sh\n",
+    "\n",
+    "# Recommended Environment Variables for optimal performance\n",
+    "export USE_XETLA=OFF\n",
+    "export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1\n",
+    "export SYCL_CACHE_PERSISTENT=1\n",
+    "```\n",
+    "\n",
+    "> **Note**\n",
+    ">\n",
+    "> For the first time that each model runs on Intel iGPU/Intel Arc A300-Series or Pro A60, it may take several minutes to compile.\n",
+    ">\n",
+    "> For other GPU type, please refer to [here](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#runtime-configuration) for Windows users, and  [here](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#id5) for Linux users.\n",
+    "\n",
+    "## `IpexLLM`\n",
+    "\n",
+    "Setting `device_map=\"xpu\"` when initializing `IpexLLM` will put the embedding model on Intel GPU and benefit from IPEX-LLM optimizations:\n",
```

Comment:
change this line to use the descriptions in llm jupyter doc. 
add the descriptions to explain prompts like in llm jupyter doc